spec/models/mets_document_spec.rb

```diff
@@ -4,28 +4,26 @@
   let(:mets_file) { Rails.root.join("spec", "fixtures", "pudl0001-4612596.mets") }
   let(:mets_file_rtl) { Rails.root.join("spec", "fixtures", "pudl0032-ns73.mets") }
   let(:mets_file_multi) { Rails.root.join("spec", "fixtures", "pudl0001-4609321-s42.mets") }
+  let(:mets_file_structure) { Rails.root.join("spec", "fixtures", "pudl0032-ns73.mets") }
```

Comment:
Do we need to load pudl0032-ns73.mets twice?  Can't we just use the existing `mets_file_rtl`?